src/App/Components/Comment/comment.holder.js

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react';
+import { connect, useSelector } from 'react-redux';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import CommentBody from './parent.comment.body';
+import actions from '../../../Redux/Actions/getAllComments';
+
+const {
+  getAllComments, postComment, updateComment, postCommentReply
+} = actions;
+
+const CommentHolder = ({
+  getAllComments: getComments,
+  postComment: createComment,
+  updateComment: newComment,
+  postCommentReply: replyComment,
+  slug
+}) => {
+  const data = useSelector(state => state.commentsReducer.All_Comments);
+  const userData = useSelector(state => state.notifications.profile);
+  const extractedData = userData.profile || '';
+  const [commentText, setComment] = useState('');
+  const [limit, setLimit] = useState(5);
+
+  /* istanbul ignore next */
```

Comment:
why are you ignoring this line @MCFrank16

Reply:
I wasn't able to test those lines so I ignored them so that the coverage won't reach on them

Reply:
@MCFrank16 don't ignore them. Let's know the accurate coverage.